labgrid/util/expect.py

```diff
@@ -22,6 +24,15 @@ def send(self, s):
         b = s
         return self.driver.write(b)
 
+    def sendcontrol(self, char):
+        """Send control character to underlying transport, e.g. Ctrl-C or Ctrl-Z."""
+        char = char.lower()
+        try:
+            ord_ = string.ascii_lowercase.index(char.lower()) + 1
```

Comment:
The second call to .lower() is redundant.

Reply:
Fixed.